soh/src/overlays/actors/ovl_En_Mag/z_en_mag.h

```diff
@@ -40,4 +41,13 @@ typedef enum {
     /* 0x04 */ MAG_STATE_POST_DISPLAY
 } EnMagGlobalState;
 
+#define dgTitleCopyrightGCTex "__OTR__objects/object_mag/gTitleCopyright19982003Tex"
+static const ALIGN_ASSET(2) char gTitleCopyrightGCTex[] = dgTitleCopyrightGCTex;
+
+#define dgTitleCopyrightN64Tex "__OTR__objects/object_mag/gTitleCopyright1998Tex"
+static const ALIGN_ASSET(2) char gTitleCopyrightN64Tex[] = dgTitleCopyrightN64Tex;
+
+#define dgTitleMQSubtitleTex "__OTR__objects/object_mag/gTitleMasterQuestSubtitleTex"
+static const ALIGN_ASSET(2) char gTitleMQSubtitleTex[] = dgTitleMQSubtitleTex;
+
```

Comment:
See comment above:
The MQ subtitle texture is already available in `object_mag.h` as `gTitleMasterQuestSubtitleTex`. No need to re-declare it.